client/src/common/organization.ts

```diff
@@ -37,3 +38,21 @@ export function formatExpirationTime(duration: Duration): Translatable {
   }
   return 'HomePage.organizationList.expiration.expired';
 }
+
+export function checkTrialAndSetExpiration(
```

Comment:
You don't need another function to get the expirationDuration, the function for that already exists: `getDurationBeforeExpiration`. You just need a function to check if a device belongs to a trial organization.
`export function isTrialOrganizationDevice(device: AvailableDevice): boolean {
  // some code here
  return isTrialOrg;
}`